packages/compiler-core/src/transforms/vIf.ts

```diff
@@ -293,7 +298,7 @@ function createChildrenCodegenNode(
     const ret = (firstChild as ElementNode).codegenNode as
       | BlockCodegenNode
       | MemoExpression
-    const vnodeCall = getMemoedVNodeCall(ret)
+    const vnodeCall = getMemoedOrOnceVNodeCall(ret)
```

Comment:
I'm wondering about the casting of `ret`. Should that also include `CacheExpression`?

I realise this would lead to type changes being needed elsewhere, but the incorrect casting may be hiding further bugs.